### ISSUE TITLE
Pytorch versions for deployment on different machines

### ISSUE THREAD

**Issue by fzoric8** [open-mmlab/mmdeploy#2283]
I'm trying to deploy `mask-rcnn` on the jetson  by using model trained on the custom data. 
Model exhibits satisfactory behavior when I check predictions on the PC I've used to train it. 
It is trained with pytorch `1.12`. 

I'm trying to deploy it on a jetson that has pytorch `1.13`. After the conversion, model doesn't predict at all. 
Conversion of the exemplary model (retinanet18) works 

**Can I use different pytorch versions for training/deployment?**

_Originally posted by @fzoric8 in https://github.com/open-mmlab/mmdeploy/discussions/2281_

**Comment by RunningLeon** [comment]
Hi, maybeyou could convert your model and test it on your PC in the first place.

**Comment by fzoric8** [comment]
Hi, there are two main reasons why I didn't do that already: 
1. I want to deploy my model on the Jetson machine, so deploying it on a PC is not an option for me 
2. I didn't want to go and setup `mmdeploy` on the PC before checking if it may be pytorch mismatch issue. 

For the time being, I'll try to train model with the same pytorch, and use same mmdetection version as one used on the jetson. 
After that, I'll try to convert `mask_rcnn` using `tools/deploy` on the host PC. 

There's this notion in my head that I can't convert and optimize a model on a different architecture (x86_64) then one I want to use it on (aarch_64). 

Maybe I wasn't clear enough, I want to train model on a PC, and then use jetson and `mmdeploy` to deploy it optimized for real-time inference on jetson.

**Comment by fzoric8** [comment]
I've managed to solve it by using same `torch`, `torchvision` and `cuda-toolkit` on the PC where network was trained. 

**Conclusion:** For successful deployment of the trained model on the another PC (jetson in this case), environment of the training PC has to be same as the environment on the deployment PC. Otherwise, it doesn't work (`mmcv` and `mmdetection`, `cuda`, `pytorch` and `torchvision` are same version).
